OneTimePassword/Keychain.swift

```diff
@@ -31,7 +121,7 @@ func addKeychainItemWithAttributes(attributes: NSDictionary) -> NSData? {
     return result as? NSData
 }
 
-func updateKeychainItemForPersistentRef(persistentRef: NSData, withAttributes attributesToUpdate: NSDictionary) -> Bool {
+internal func updateKeychainItemForPersistentRef(persistentRef: NSData, withAttributes attributesToUpdate: NSDictionary) -> Bool {
```

Comment:
Length Violation: Line should be 100 characters or less: currently 130 characters